public-assets/css/rte-content.css

```diff
@@ -0,0 +1,46 @@
+/* TODO => can we import uui colors? */
```

Comment:
Yes, they should all be available even though IntelliSense might suggest otherwise. They are loaded on the first line of `app.ts`.